package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@reactivex/rxjs",
-  "version": "6.0.0-alpha.3",
+  "version": "6.0.0-alpha.2",
```

Comment:
Oops. :D you probably didn't mean to do this.

Reply:
yeah I think oj had something merged in, so I just took incoming changes.  Fixed.